src/access/ownable/ownable.cairo

```diff
@@ -83,11 +100,72 @@ mod OwnableComponent {
         TContractState, +HasComponent<TContractState>
     > of interface::IOwnableCamelOnly<ComponentState<TContractState>> {
         fn transferOwnership(ref self: ComponentState<TContractState>, newOwner: ContractAddress) {
-            self.transfer_ownership(newOwner);
+            Ownable::transfer_ownership(ref self, newOwner);
         }
 
         fn renounceOwnership(ref self: ComponentState<TContractState>) {
-            self.renounce_ownership();
+            Ownable::renounce_ownership(ref self);
+        }
+    }
+
+    /// Adds support for two step ownership transfer.
+    #[embeddable_as(OwnableTwoStepImpl)]
+    impl OwnableTwoStep<
```

Comment:
This impl should be above the OwnableCamelOnly on top. the order should be:

- Ownable
- OwnableCamelOnly
- OwnableTwoStep
- OwnableTwoStepCamelOnly

Reply:
The order is like that - Ownable, OwnableCamelOnly, OwnableTwoStep, OwnableTwoStepCamelOnly.

Reply:
@martriay yes you are right, for some reason I suggested the previous order we were using, and the the current one, MB. cc @milancermak